ste/mgr-JobPartTransferMgr.go

```diff
@@ -345,7 +345,11 @@ func (jptm *jobPartTransferMgr) Info() TransferInfo {
 				 * we can have 4 blocks in core, waiting for a disk or n/w operation. Any higher block size would *sort of*
 				 * serialize n/w and disk operations, and is better avoided.
 				 */
-				blockSize = sourceSize / common.MaxNumberOfBlocksPerBlob
+				if (sourceSize % common.MaxNumberOfBlocksPerBlob == 0) {
```

Comment:
Nit: [Ceil function](https://golangbyexample.com/ceil-number-golang/) perhaps?

Reply:
I believe this stems from the comment here: https://github.com/Azure/azure-storage-azcopy/pull/1745/commits/36311199db508cb92b424206ec7b1c7996ca66e8#r858641439

If we are to compare what @nitin-deamon mentioned as a reply there versus this line of code, then in my opinion would be preferable to avoid typecasting of int-float & vice-versa.